userspace/libpman/include/libpman.h

```diff
@@ -25,6 +25,10 @@ extern "C"
 {
 #endif
 
+	/* Forward decleare them */
```

Comment:
Not needed, but just in case you wanted to fix some types, `declare` and also `Return a `scap_stats_v2` ...` below

Reply:
sure i will do it, thanks!